package.json

```diff
@@ -191,6 +191,6 @@
     "typescript": "^4.2.4"
   },
   "engines": {
-    "node": ">=10.13.0"
+    "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
```

Comment:
Why these versions, and not latest versions?

Reply:
I follow the guide: https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

> Update the "engines" field in package.json to Node.js 12: `"node": "^12.20.0 || ^14.13.1 || >=16.0.0"`.

Since Node 12.20.0, pure ESM has been supported.

See also: https://github.com/stylelint/stylelint/issues/5205#issuecomment-833146267